pnrm/plugins/swfwlicense/licensemgr/pano.go

```diff
@@ -0,0 +1,137 @@
+package licensemgr
+
+import (
+	"encoding/xml"
+	"github.com/PaloAltoNetworks/pango/namespace"
+	"github.com/PaloAltoNetworks/pango/util"
+)
+
+// Panorama is the client.Panorama namespace.
+type Panorama struct {
+	ns *namespace.Plugin
+}
+
+// GetBootstrapParams for specific licensemgr and returns a string of the bootstrap parameters and an error if needed
+func (c *Panorama) GetBootstrapParams(licensemgr string) (string, error) {
+	type requestStruct struct {
+		XMLName    xml.Name `xml:"request"`
+		LicenseMgr string   `xml:"plugins>sw_fw_license>bootstrap-parameters>license-manager"`
+	}
+
+	type resultStruct struct {
+		BootstarpParams string `xml:"result>bootstrap-parameters"`
+	}
+
+	var result resultStruct
+	
+	c.ns.Client.LogOp("(op) request license manager bootstrap parameters for " + licensemgr )
+	body, err := c.ns.Client.Op(requestStruct{LicenseMgr: licensemgr}, "", nil, nil)
+	if err != nil {
+		return "", err
+	} else {
+		err = xml.Unmarshal(body, &result)
+		if err != nil {
+			return "", err
+		} else {
+			return result.BootstarpParams, nil
+		}
+	}
```

Comment:
Typically you just pass in the struct you want the answer unmarshalled into and let `Op()` do it for you.  If the result of this `op` command doesn't require special handling, then just pass in `&result` and let `Op()` do the work.

Reply:
Didn't realize the `op` command did, that so changed that too.